docs/latest/modules/en/pages/use/dashboards/dashboard-variables.adoc

```diff
@@ -0,0 +1,103 @@
+:revdate: 2025-11-24
+:page-revdate: {revdate}
+:description: Use variables to create dynamic dashboards.
+
+By configuring variables for a dashboard, you can create dashboards with dynamic widgets. Variables enable the reuse of a single dashboard definition for different use-cases. 
+
+== Viewing dashboard variables
+In the top navigation bar, use the dropdown menu next to the "Save..." button, and click "Edit dashboard variables".
+
+// TODO STAC-22767 Add a screenshot
+
+=== Built-in variables
+There are variables that are built-in to {stackstate-product-name}. The names of these variables are reserved, and you cannot give your own variables the same names.
+
+To view all built-in variables, click the accordion titled "SUSE Observability PromQL variables".
+
+== Adding a new variable
+
+. Open the "Edit dashboard variables" drawer. 
+. Click the "Add variable" button to open a second drawer where you can configure the new variable.
+
+// TODO STAC-22767 Add a screenshot
+
+The configuration fields for a variable are as follows:
+
+|===
+| Field name | Description
+
+| Name
+| The name identifier of the variable. This is the name that you can <<using-variables, use in compatible fields>>.
+
+| Display label
+| A friendly name for the variable that is shown on the dashboard when <<selecting-variables, selecting the variable>>. This helps you identify the variable.
+
+| Description
+| A description to understand the purpose for the variable. This description is shown in the "Edit dashboard variables" drawer.
+
+| Type
+| The type of the variable.
+|===
+
+== Variable types
+=== Text variable
+
+A text variable contains simple text. You can set an initial value and make the variable read-only.
+
+=== List variable
+
+A list variable has possible values which are defined by the "Source" field. Each source type provides additional fields which allow you to narrow down the possible values for the variable.
+
+|===
+| Source type | Source fields
+
+| Metric label names
+| Enter zero or more series selectors to refine which metric label names are present.
+
+| Metric label values
+| Enter a metric label name, and zero or more series selectors to refine which metric label values are present.
+
+| Metric PromQL
+| Enter a custom PromQL query, and a label name. The resulting possible values are the values for the given label name.
+
+| Static list
+| Enter one or more label and value pairs to construct a static list of possible values.
+|===
+
+As you change the source type and their related fields, the preview on the right-hand-side of the drawer will update with the new possible values.
+
+
+==== Common list variable settings
+
+All list variable sources also have the following common settings:
+
+|===
+| Field name | Description
+
+| Sort
+| Sort the values in the list. For example, alphabetically or numerically.
+
+| Allow multiple values
+| Allow users to select more than one value at a time.
+
+| Enable "Include Everything" option
```

Comment:
For both the `multiple` and the `everything`  option we need to document that it produces a regex during interpolation so it should be used with the Promql regex matcher: `=~`.